### PR TITLE
JSdoc style for prop types

### PR DESCRIPTION
**Pull request by jynnie**
Want feedback on this type of comment for documentation

### CODE REVIEW

File: client/src/components/modules/NewPost.js [hunk 1]
```diff
@@ -2,6 +2,14 @@ import React, { Component } from "react";
...
 import "./NewPost.css";
 
+/**
+ * New Post is a parent component for all input components
+ *
+ * Proptypes
+ * defaultText: (string) is the placeholder text
+ * storyId: (string) optional prop, used for comments
+ * onSubmit: (function) triggered when this post is submitted, takes {storyId, value} as parameters
+ */
```
Comment (jynnie) [inline client/src/components/modules/NewPost.js]: This is an example of what docs would look like in not JSDoc style

**Comment by jynnie** [comment]
Nov-12-2019 14-20-21
An argument for using the JSDoc style is that VSCode recognizes those type definitions and renders them pretty. But we should also think about how ugly our prop types are (see StoryData, StoryObject).

File: client/src/components/modules/Card.js [hunk 2]
```diff
@@ -5,6 +5,15 @@ import { get } from "../../utilities";
 
 import "./Card.css";
 
+/**
+ * Card is a component for displaying content like stories
+ *
+ * Proptypes
+ * @param {string} _id of the comment
+ * @param {string} creator_id
```
Comment (cory2067) [inline client/src/components/modules/Card.js]: wasn't this field removed in 30d062d24220a51680eb25de4d7e3db821a49a95

**Comment by jynnie** [comment]
Merged into noauth

**Pull request closed without merging**